### DISCUSSION TITLE
Can this be implemented in a triton inference server??

### DISCUSSION THREAD

**Discussion by mishmashlabs** [RizwanMunawar/yolov7-object-tracking#24] [Ideas]
I'm aware yolov7 can be deployed in a triton server by building an engine from the onnx file,

Can the sort tracker be implemented similarly?

**Comment by RizwanMunawar** [comment]
@mishmashlabs! Yes, you can deploy it, but for this, you need to add a tracker in the triton server backend code files. I haven't experience with that, so not sure where you can take help.
